pymeasure/instruments/hp/hp8753e.py

```diff
@@ -0,0 +1,472 @@
+        """Restart sweep averaging."""
+        self.write("AVERREST")
+
+    def emit_beep(self):
+        """Make the VNA emit a beep"""
+        self.write("EMIB")
+
+    power = Instrument.control(
+        "POWE?",
+        "POWE%.3e",
+        """Control the output RF power of the instrument's active port. (float).
+        Property is dynamic and allows remapping min/max limits.
+        """,
+        cast=float,
+        validator=strict_range,
+        dynamic=True,
+        values=[-70, 10],
+    )
+
+    output_enabled = Instrument.control(
+        "SOUP?",
+        "SOUP%s",
+        """Control RF Output State (boolean)""",
+        cast=bool,
+        map_values=True,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0, "ON": 1, "OFF": 0},
+    )
+
+    trigger_hold = Instrument.control(
+        "HOLD?",
+        "%s",
+        """Control Sweep Scan Trigger State (boolean)""",
+        cast=str,
+        set_process=lambda x: "HOLD" if x else "CONT",
+        get_process=lambda x: x == "1",
+    )
+
+    trigger_continuous = Instrument.control(
+        "CONT?",
+        "%s",
+        """Control Sweep Scan Trigger State (boolean)""",
+        cast=str,
+        set_process=lambda x: "CONT" if x else "HOLD",
+        get_process=lambda x: x == "1",
+    )
+
+    scan_points = Instrument.control(
+        "POIN?",
+        "POIN%d",
+        f"""Control the number of points used for a scan sweep.
+        From the set {SCAN_POINT_VALUES}""",
+        cast=lambda x: int(float(x)),
+        validator=strict_discrete_set,
+        values=SCAN_POINT_VALUES,
+    )
+
+    id = Instrument.measurement(
+        "*IDN?",
+        """Get the identification of the instrument (list of strings)""",
+        cast=str,
+    )
+
+    sn = Instrument.measurement(
+        "OUTPSERN",
+        """Get the serial number for the instrument""",
+    )
+
+    options = Instrument.measurement(
+        "OUTPOPTS",
+        """Get the installed options for the instrument""",
+    )
+
+    @property
+    def manu(self):
+        """Get the manufacturer of the instrument."""
+        if self._manu == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._manu
+
+    @property
+    def model(self):
+        """Get the model of the instrument."""
+        if self._model == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._model
+
+    @property
+    def fw(self):
+        """Get the firmware of the instrument."""
+        if self._fw == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._fw
+
+    def set_fixed_frequency(self, frequency):
+        """Set the sweep to be a fixed frequency in Hz."""
+        self.start_frequency = frequency
+        self.stop_frequency = frequency
+        self.scan_points = 3
+
+    @property
+    def measuring_parameter(self):
+        """Get the active Scattering or Measuring Parameter being measured.
```

Comment:
Beware, the getter's docstring will be used for the whole property!

Therefore, you should use "Control" here and add also the content of the setter's docstring, which will be invisible later (and you can remove it).

Reply:
Changed the docstring for measuring_parameter getter to use 'Control' and removed the docstring to the setter.